src/visitors/transpileCssProp.js

```diff
@@ -120,7 +120,13 @@ export default t => (path, state) => {
     }, [])
   }
 
-  program.node.body.push(
+  let parent = elem
+
+  while (!t.isProgram(parent.parentPath)) {
```

Comment:
I assume this should be a safe thing to do since before we were basically doing the same thing but an append instead of a sibling prepend to the node tree that we're currently working on.